canvas_course_admin_tools/settings/base.py

```diff
@@ -27,27 +27,30 @@
 # Application definition
 
 INSTALLED_APPS = (
+    'async',
+    'canvas_course_admin_tools',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'django_auth_lti',
-    'icommons_common',
-    'lti_permissions',
-    'async',
     'django_rq',
-    'icommons_ui',
     'djangular',
-    'canvas_course_admin_tools',
+    'icommons_common',
+    'icommons_common.monitor',
+    'icommons_ui',
     'isites_migration',
+    'lti_permissions',
+    'manage_people',
+    'manage_sections',
 )
 
 MIDDLEWARE_CLASSES = (
     'djangular.middleware.DjangularUrlMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
This reminds me that it was on my to-do list to figure out how to integrate this with our apps, plus the `XFrameOptionsMiddleware`. But that can be tech debt / research.